rlclientlib/live_model_impl.cc

```diff
@@ -77,7 +77,7 @@ namespace reinforcement_learning {
       RETURN_IF_FAIL(reset_action_order(response));
     }
 
-    RETURN_IF_FAIL(_ranking_logger->log(event_id, context, flags, response, status, _learning_mode));
+    RETURN_IF_FAIL(_ranking_logger->log(context, flags, response, status, _learning_mode));
```

Comment:
Trying to read this PR with the proposal in the OneNote. Let me know if I missed this - 

How is the V2 protocol enabled from config/live_model? Here - not seeing a _ranking_logger change when it gets created.

Reply:
this will come in a future PR

Reply:
actually config parameter was introduced here:
https://github.com/VowpalWabbit/reinforcement_learning/pull/163